src/dev/pages/text-field/text-field.ejs

```diff
@@ -1,21 +1,36 @@
-<forge-text-field id="text-field">
-  <!-- Leading -->
-  <forge-icon slot="leading" name="person"></forge-icon>
+<div class="vert">
+  <div>
+    <h3 class="forge-typography--heading2">Default</h3>
+    <forge-text-field>
+      <forge-icon slot="start" name="person"></forge-icon>
+      <label>First name</label>
+      <input type="text" id="text-field-input" />
+      <textarea id="text-field-textarea"></textarea>
+      <forge-icon slot="end" name="info_outline"></forge-icon>
+      <forge-icon-button slot="accessory" density="medium">
+        <forge-icon name="more_vert"></forge-icon>
+      </forge-icon-button>
+      <span slot="support-text-start" slot="support-text-start">Please enter your first name</span>
+      <span slot="support-text-end" slot="support-text-end"></span>
+    </forge-text-field>
+  </div>
 
-  <!-- Label and input -->
-  <label for="text-field-input">First name</label>
-  <input type="text" id="text-field-input" />
+  <div>
+    <h3 class="forge-typography--heading2">Multiple inputs</h3>
+    <forge-text-field>
+      <label>First and last name</label>
+      <input type="text" />
+      <input type="text" />
```

Comment:
For multiple inputs in text fields, I wonder if we should manage the tabindex of the second input to only be keyboard accessible if the first input has a value? We could add a property/attribue to control whether that happens as well? Additionally, if you attempt to click into that second input in this scenario, it could also immediately move focus to the first input as well.

Maybe add a todo item to the board for this so we can revisit if we don't want to hold this up right now.

Reply:
Let's split this out